spec/controllers/api/v1/blood_pressures_controller_spec.rb

```diff
@@ -1,24 +1,21 @@
 require 'rails_helper'
 
 RSpec.describe Api::V1::BloodPressuresController, type: :controller do
-  describe 'POST sync: send data from device to server;' do
+  let(:model) { BloodPressure }
 
-    describe 'creates new blood pressures' do
+  let(:build_payload) { lambda { build_blood_pressure_payload } }
+  let(:build_invalid_payload) { lambda { build_invalid_blood_pressure_payload } }
+  let(:update_payload) { lambda { |blood_pressure| updated_blood_pressure_payload blood_pressure } }
 
-      it 'returns 400 when there are no blood pressures in the request' do
-        post(:sync_from_user, params: { blood_pressures: [] })
-        expect(response.status).to eq 400
-      end
+  let(:number_of_schema_errors) { 3 }
```

Comment:
I don't know what this is for?

Reply:
We have renamed this to `number_of_schema_errors_in_invalid_payload`. `invalid_payload` is defined above, and this defines the number of schema errors in that.